content/F2014/F14-media-sarah-sharp.md

```diff
@@ -250,226 +249,221 @@ read from the USB devices.  And they do that through an interface to the Linux
 kernel.
 
 So USBFS is basically that interface.  It is a bunch of system calls.  The
-system calls, types it uses is actually Ioctls. And ioctls are like this really
+type of system calls that it uses is actually Ioctls. Ioctls are like this really
```

Comment:
`ioctls` should be in teletype font, like this:

```
`ioctl`s
```

Since they are a system call and properly referred to in lowercase.